### PR TITLE
[kube-prometheus-stack] Update dependencies and dashboard

### PR DESCRIPTION
**Pull request by mrueg**
#### What this PR does / why we need it:
Updates grafana, node-exporter dependencies as well as syncs dashboards

#### Checklist

- [x] [DCO](https://github.com/prometheus-community/helm-charts/blob/main/CONTRIBUTING.md#sign-off-your-work) signed
- [x] Chart Version bumped
- [x] Title of the PR starts with chart name (e.g. `[prometheus-couchdb-exporter]`)

### CODE REVIEW

File: charts/kube-prometheus-stack/Chart.yaml [hunk 1]
```diff
@@ -18,7 +18,7 @@ name: kube-prometheus-stack
 sources:
   - https://github.com/prometheus-community/helm-charts
   - https://github.com/prometheus-operator/kube-prometheus
-version: 16.14.1
+version: 16.14.2
```
Comment (mrueg) [inline charts/kube-prometheus-stack/Chart.yaml:21]: node_exporter major bump is mainly about helm2 compatibility which this chart is unaffected by, thus not raising minor here.

**Review by Xtigyro — DISMISSED** [review]
Thanks for the contribution, @mrueg! 🥇 

LGTM.

**Review by Xtigyro — COMMENTED** [review]
@mrueg It appears the chart cannot be installed properly.

**Comment by mrueg** [comment]
> @mrueg It appears the chart cannot be installed properly.

Yes I'm looking into it. It could be related to https://github.com/prometheus-community/helm-charts/pull/1181

**Comment by mrueg** [comment]
@Xtigyro this was broken by a change in grafana and got fixed through https://github.com/grafana/helm-charts/pull/575

**Review by Xtigyro — CHANGES_REQUESTED** [review]
Prometheus minor version is bumped. Please let's do so with the chart version.

**Comment by mrueg** [comment]
> Prometheus minor version is bumped. Please let's do so with the chart version.

Done!

**Review by Xtigyro — APPROVED** [review]

**Pull request merged**